mmpose/apis/inference.py

```diff
@@ -73,6 +79,12 @@ def _xywh2xyxy(bbox_xywh):
     bbox_xyxy = bbox_xywh.copy()
     bbox_xyxy[:, 2] = bbox_xyxy[:, 2] + bbox_xyxy[:, 0] - 1
     bbox_xyxy[:, 3] = bbox_xyxy[:, 3] + bbox_xyxy[:, 1] - 1
+
+    # Select bboxes by score threshold
+    if bbox_thr:  # bbox_thr not None
```

Comment:
Use the statement in L62.
When bbox_thr == 0.0, it will be skipped by this condition.